KBaseMetagenomes.spec

```diff
@@ -3,16 +3,20 @@
 
 module KBaseMetagenomes {
 
-    /*
-    Data type for binned contigs:
-
-    typedef struct {
-        ...
-
-    } BinnedContigs;
+    typedef string bin_id;
+    typedef string contig_id;
 
+    /*
+     Reference to an assembly object
+     @id ws KBaseGenomeAnnotations.Assembly
     */
+    typedef string contig_obj_ref;
 
+    typedef structure {
+        list<bin_id> bin_ids;
+        mapping<bin_id, list<contig_id>> bin2contig_id;
+        mapping<contig_id, contig_obj_ref> contig_id2contig_ref;
```

Comment:
Well, I'm thinking instead of this map, we may just want one field that references the one Assembly the contigs came from, rather than a separate Assembly ref per contig.